package.json

```diff
@@ -19,14 +19,19 @@
   "dependencies": {
     "@svgr/webpack": "^5.5.0",
     "gatsby": "^2.26.1",
+    "gatsby-plugin-react-helmet": "^3.7.0",
     "gatsby-plugin-react-svg": "^3.0.0",
     "gatsby-plugin-svgr": "^2.0.2",
     "howler": "^2.2.1",
     "lodash": "^4.17.20",
     "markdown-to-jsx": "^7.1.0",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
+<<<<<<< HEAD
+    "react-helmet": "^6.1.0",
+=======
     "react-tooltip": "^4.2.11",
+>>>>>>> origin/main
```

Comment:
This needs fixing

Reply:
probably add them both and run `yarn` to generate the lock file again

Reply:
whoops done :-)